runtime/oti/j9vm.hdf

```diff
@@ -267,6 +267,15 @@ typedef UDATA (* lookupNativeAddressCallback)(struct J9VMThread *currentThread,
 		<struct>J9VMAcquireVMAccessEvent</struct>
 		<data type="struct J9VMThread*" name="currentThread" description="current thread" />
 	</event>
+	
+	<event>
+		<name>J9HOOK_VM_RELEASEVMACCESS</name>
+		<description>
+				Triggered when a thread is about to release VM access.
```

Comment:
Probably worth documenting that the hook is called after access is released (which is the opposite of what the current comment implies).